### PR TITLE
Update Checking Security Vulnerabilities doc part

### PR DESCRIPTION
**Pull request by 94noni**
Friendly ping @fabpot 
Ref https://github.com/symfony-cli/symfony-cli/issues/9

### CODE REVIEW

File: setup.rst [hunk 1]
```diff
@@ -225,26 +225,15 @@ require --no-unpack ...`` option to disable unpacking.
...
-
-    $ symfony check:security
-
-A good security practice is to execute this command regularly to be able to
+You can also execute this process regularly to be able to
 update or replace compromised dependencies as soon as possible. The security
 check is done locally by fetching the public `PHP security advisories database`_,
 so your ``composer.lock`` file is not sent on the network.
 
-.. tip::
```
Comment (fabpot) [inline setup.rst]: This tip is also valid for the security checker, so it should be kept IMHO.

**Review by fabpot — CHANGES_REQUESTED** [review]

File: setup.rst (hunk 1 above)
Comment (94noni) [reply setup.rst]: added back

File: setup.rst [hunk 2]
```diff
@@ -225,15 +225,11 @@ require --no-unpack ...`` option to disable unpacking.
 Checking Security Vulnerabilities
 ---------------------------------
 
-The ``symfony`` binary created when you `install Symfony CLI`_ provides a command to
-check whether your project's dependencies contain any known security
-vulnerability:
+A good and recommended security practice is to check from time to time whether your
+project's dependencies contain any known security vulnerability. You can leverage the
```
Comment (fabpot) [inline setup.rst:229]:
```suggestion
project's dependencies contain any known security vulnerabilities. You can leverage the
```

**Review by fabpot — APPROVED** [review]
Small typo

File: setup.rst (hunk 2 above)
Comment (javiereguiluz) [reply setup.rst:229]: Fixed while merging. Thanks!

**Comment by javiereguiluz** [comment]
Thanks Antoine!

**Pull request merged**